### PR TITLE
Update roles

### PR DESCRIPTION
**Pull request by grst**
Update the roles document. 

 - Add description of management committee
 - Add description of advisory board

### CODE REVIEW

**Comment by grst** [comment]
Don't know how much more you'd want to add here?

File: in_progress/roles.md [hunk 1]
```diff
@@ -70,11 +70,8 @@ Primary maintainers of new core packages, who are not already core developers, w
 ## Steering Council
 
 The Steering Council (SC) members are core team members who have additional responsibilities to ensure the smooth running of the project.
-SC members are expected to participate in strategic planning, approve changes to the governance model, and make decisions about funding granted to `scverse` itself.
-(Funding to community members is theirs to pursue and manage). 
-The purpose of the SC is to ensure smooth progress from the big-picture perspective.
-Changes that impact the full project require analysis informed by long experience with both the project and the larger ecosystem.
-When the core team (including the SC members) fails to reach such a consensus in a reasonable timeframe, the SC is the entity that resolves the issue.
+SC members are expected to participate in strategic planning, approve changes to the governance model, and make decisions about funding granted to `scverse` itself (funding to community members is theirs to pursue and manage). Changes that impact the full project require analysis informed by long experience with both the project and the larger ecosystem.
```
Comment (Zethson) [inline in_progress/roles.md:73]:
```suggestion
SC members are expected to participate in strategic planning, approve changes to the governance model, and make decisions about funding granted to `scverse` itself (funding to community members is theirs to pursue and manage). Changes that impact the full project require analysis informed by extensive experience with both the project and the larger ecosystem.
```

**Review by Zethson — COMMENTED** [review]
Concise and to the point.

Think that having it rather short is good initially because we can add more details later if required.

:+1:

**Review by ivirshup — APPROVED** [review]
Looks great

**Pull request merged**